modules/ROOT/pages/manage-proxy-self.adoc

```diff
@@ -0,0 +1,69 @@
+= Manage Proxies on Runtime Fabric on Self-Managed Kubernetes
```

Comment:
Same comment as above. Keep it consistent to either `Used by` or `on`.

Reply:
Fixed.